em-seq.nf

```diff
@@ -151,7 +151,7 @@ process mergeAndMarkDuplicates {
 
         shell:
         '''
-        MethylDackel extract --methylKit --OT 0,0,0,95 --OB 0,0,5,0 -@ !{task.cpus} --CHH --CHG -o !{library} !{genome} !{md_file}
+        MethylDackel extract --methylKit --nOT 0,0,0,5 --nOB 0,0,5,0 -@ !{task.cpus} --CHH --CHG -o !{library} !{genome} !{md_file}
```

Comment:
Do these produce the same results?
I can't quite tell from the documentation

Reply:
`It should`. But I will test it in small bams 100bp long to be sure!

Reply:
Tested in old data from Chaythanya, with reads 100bp long.
New and old arguments show 100% of CpGs with same level of methylation. 
As a negative control, I tested the command without trimming and 9% of the CpGs show different levels of methylation.

Reply:
Thanks for checking!

I don't know why we didn't pick that option.